### PR TITLE
Added an option to hide numpad for phones with a physical numpad

### PR DESCRIPTION
**Pull request by ronniedroid**
I added an option in the settings to hide the numbers in the, it only keeps the *, + and  #, keys.

Please note that I kept the + button as it is necessary, and the * and # I kept for style.  

This is a useful option for people with phones that have a physical dialpad. 

Possible improvements that are above my skills:
- Hide the * + and # keys too, and make long pressing 0 on the physical dialpad input a "+".
- Make the Physical dial button work as expected (Dial the inputted number)

### CODE REVIEW

File: app/src/main/res/layout/activity_settings.xml [hunk 1]
```diff
@@ -227,6 +227,22 @@
...
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@drawable/ripple_background">
+
+                    <com.simplemobiletools.commons.views.MyAppCompatCheckbox
+                        android:id="@+id/settings_hide_dialpad_numbers"
+                        style="@style/SettingsCheckboxStyle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:text="Hide dialpad numbers" />
```
Comment (tibbi) [inline app/src/main/res/layout/activity_settings.xml:242]: oh cmon, move this into the string file of all languages (in english) and fetch it from there

File: app/src/main/res/layout/dialpad.xml [hunk 2]
```diff
@@ -18,6 +18,7 @@
         android:layout_marginStart="@dimen/activity_margin"
         android:background="?attr/selectableItemBackgroundBorderless"
         android:contentDescription="@string/one"
+        android:visibility="visible"
```
Comment (tibbi) [inline app/src/main/res/layout/dialpad.xml:21]: no need to set Visible everywhere, that is the default one

File: app/src/main/res/layout/activity_settings.xml (hunk 1 above)
Comment (ronniedroid) [reply app/src/main/res/layout/activity_settings.xml:242]: will do

File: app/src/main/res/layout/dialpad.xml (hunk 2 above)
Comment (ronniedroid) [reply app/src/main/res/layout/dialpad.xml:21]: I see, I will remove it.

**Comment by ronniedroid** [comment]
I removed the unnesecarry visible atteributes and moved the "Hide dialpad numbers" to the strings wills of all the language, (and translated the Arabic one).

**Comment by tibbi** [comment]
is the keyboard usually hidden by default in other apps?

**Comment by ronniedroid** [comment]
> is the keyboard usually hidden by default in other apps?

I don't understand what you mean

**Comment by tibbi** [comment]
If you check Google Phone or some other Dialer apps with the device that has numpad, do their built in dialpads hide the numbers like you are suggesting in this app?

**Comment by ronniedroid** [comment]
No, they don't, no one thinks of phones with keys. People with such phones are stuck with closed source OEM provided ones. Are you thinking of making this option on by default on such devices?

**Comment by tibbi** [comment]
not sure, so far nobody requested it. There are just a couple users with such devices I guess

**Comment by ronniedroid** [comment]
Not many, but Simple apps are the goto to most of them. Besides, an option in the settings won't hurt, right?

**Comment by tibbi** [comment]
sure, Im considering it

**Comment by tibbi** [comment]
it crashes at long pressing Plus with the new setting enabled

**Comment by ronniedroid** [comment]
The  crashing on long pressing the Plus button was fixed.

**Comment by tibbi** [comment]
should be good, thanks

**Pull request merged**